osmnx/save_load.py

```diff
@@ -248,7 +249,10 @@ def load_graphml(filename, folder=None):
     # convert numeric node tags from string to numeric data types
     log('Converting node and edge attribute data types')
     for _, data in G.nodes(data=True):
-        data['osmid'] = int(data['osmid'])
+        if node_type == int:
+            data['osmid'] = int(data['osmid'])
+        else:
+            data['osmid'] = ast.literal_eval(data['osmid'])
```

Comment:
One last question: I see that your logic either converts `data['osmid']` to `int` if `node_type==int`, or it does a `literal_eval` on `data['osmid']`... however the docstring says "node_type : type (Python type (default: int)) – Convert node ids to this type". So if a user passes `node_type=str`, they would expect it to convert `data['osmid']` to a string, but your current logic would `literal_eval` it instead, right?

Reply:
Thanks
Uou are right, I change it, to be more consistent